### PR TITLE
Cleans up some of the code for unhusking people with chems

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

Replaces all mentions of "TRESHOLD_UNHUSK" with "T**H**RESHOLD_UNHUSK" (seriously, guys?).

Updates the description of synthflesh to reflect what it actually does more accurately (a percentage in the description was wrong because of a stealth revert that oranges made).

Changes the name of the variable "Carbies" to "carbies", because the capitalization of that variable's name was annoying me.

Using a patch to bring the amount of synthflesh in a burnt husk's body to 100u or more now unhusks it immediately, instead of requiring an additional application of synthflesh after that.

I also tweaked the description of the text that's displayed when a corpse is unhusked by synthflesh a little bit.

## Why It's Good For The Game

See above.

## Changelog
:cl: ATHATH
fix: Using a patch to bring the amount of synthflesh in a burnt husk's body to 100u or more now unhusks it immediately, instead of requiring an additional application of synthflesh after that.
fix: Updates the description of synthflesh to reflect what it actually does more accurately. The amount of tox damage dealt is 66% of the amount of brute and burn damage healed, not 75%.
/:cl:

### CODE REVIEW

**Comment by Angustmeta** [comment]
I don't think Oranges' revert of the Synthflesh buff was actually stealthy. A proper revert PR was made. I just failed to notice it.

**Comment by ATH1909** [comment]
> I don't think Oranges' revert of the Synthflesh buff was actually stealthy. A proper revert PR was made. I just failed to notice it.

Ah, okay. I've edited the changelog to account for this.

**Comment by ATH1909** [comment]
Should I add "Can also be used to unhusk corpses under certain conditions." to the descriptions of synthflesh and rezadone?

**Comment by Krysonism** [comment]
Power creep

We shouldn't trivialize husking further, it is meant to be a serious hindrance to revival.

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 1]
```diff
@@ -403,26 +403,25 @@
...
 			if(show_message)
-				to_chat(Carbies, "<span class='danger'>You feel your burns and bruises healing! It stings like hell!</span>")
-			SEND_SIGNAL(Carbies, COMSIG_ADD_MOOD_EVENT, "painful_medicine", /datum/mood_event/painful_medicine)
-			//Has to be at less than TRESHOLD_UNHUSK burn damage and have 100 isntabitaluri before unhusking. Corpses dont metabolize.
-			if(HAS_TRAIT_FROM(M, TRAIT_HUSK, "burn") && Carbies.getFireLoss() < TRESHOLD_UNHUSK && Carbies.reagents.has_reagent(/datum/reagent/medicine/C2/instabitaluri, 100))
-				Carbies.cure_husk("burn")
-				Carbies.visible_message("<span class='nicegreen'>With most of the burnt off flesh replaced, [Carbies] looks a lot healthier.</span>")
+				to_chat(carbies, "<span class='danger'>You feel your burns and bruises healing! It stings like hell!</span>")
+			SEND_SIGNAL(carbies, COMSIG_ADD_MOOD_EVENT, "painful_medicine", /datum/mood_event/painful_medicine)
+			if(HAS_TRAIT_FROM(M, TRAIT_HUSK, "burn") && carbies.getFireLoss() < THRESHOLD_UNHUSK && reac_volume >= 40)
```
Comment (kriskog) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]:
```suggestion
			if(HAS_TRAIT_FROM(M, TRAIT_HUSK, "burn") && Carbies.getFireLoss() < TRESHOLD_UNHUSK && (Carbies.reagents.get_reagent_amount(/datum/reagent/medicine/C2/instabitaluri) + reac_volume >= 100))
```

Getting 100u synthflesh is trivial, this fixes the issue where you cant splash husks to fix them.

**Review by kriskog — CHANGES_REQUESTED** [review]

**Comment by AdamElTablawy** [comment]
husking is very easy though, one full laser clip does it

100u synthflesh every time is expensive to make and produce

**Comment by kriskog** [comment]
Its supposed to be an annoying thing to fix. The threshold for getting husked was increased before cloning removal, and plumbing can supply an infinite amount of synthflesh easily enough

**Pull request merged**